pkg/config/config.go

```diff
@@ -1205,7 +1205,11 @@ func InitConfig(config Config) {
 
 	// Datadog security agent (runtime)
 	config.BindEnvAndSetDefault("runtime_security_config.enabled", false)
-	config.BindEnvAndSetDefault("runtime_security_config.socket", "/opt/datadog-agent/run/runtime-security.sock")
+	if runtime.GOOS == "windows" {
+		config.BindEnvAndSetDefault("runtime_security_config.socket", "localhost:3334")
```

Comment:
Linux has had issues properly resolving `localhost` in some instances (caused by IP version issues/resolve.conf/etc). Can we assume that Windows won't have those issues?

PS: What prevents us from using named pipes here for Windows platform?

Reply:
answer to both questions... 
1) while I haven't verified recently, historically windows guaranteed that if you looked up "localhost" you'd get 127.0.0.1.  I think it's safe

2) nothing prevents us from using named pipes.  The feature itself is still WIP, and am trying to get something so the cross-team work can continue.  It is part of the plan to have this (and other places) migrate to named pipes.